skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -178,7 +177,7 @@ def _nl_means_denoising_2d(image, int s=7, int d=13, float h=0.1):
     cdef int row_start, row_end, col_start, col_end
     cdef int row_start_i, row_end_i, col_start_j, col_end_j
     cdef IMGDTYPE [::1] new_values = np.zeros(n_ch).astype(np.float32)
-    cdef IMGDTYPE [:, :, ::1] padded = np.ascontiguousarray(util.pad(image,
+    cdef IMGDTYPE [:, :, ::1] padded = np.ascontiguousarray(np.pad(image,
```

Comment:
I'm in favour of Josh's point. As we aren't speaking about removing the functions yet (at least, while `numpy` has no own version of `crop`), we should expose them to facilitate their usage for both users and developers. Although, my opinion would be to make this happen in another place (e.g. `Novice guide`), and to refactor the comment I mentioned because _it is not strictly correct_ (an image is padded in `_nl_means_denoising.pyx` with `np.pad`, not `skimage.util.pad`).
In any case, people spend much more time reading the API docs on the website than the source code.